geomesa-core/src/test/scala/org/locationtech/geomesa/core/data/ComplexFeatureTest.scala

```diff
@@ -35,6 +35,8 @@ import scala.collection.JavaConverters._
 @RunWith(classOf[JUnitRunner])
 class ComplexFeatureTest extends Specification with TestWithDataStore {
 
+  sequential
```

Comment:
I'll add a comment. This was due to that issue I mentioned to you about how a mock connector returns the exact same byte[] for multiple scans, which causes kryo to occasionally get thread conflicts which causes OOM errors when it tries to read the length of an array to allocate memory and it ends up getting some huge number... I'm pretty sure this is only a test problem though...